app/src/main/java/org/wikipedia/activitytab/timeline/TimelineModule.kt

```diff
@@ -138,6 +136,43 @@ fun TimelineModule(
     }
 }
 
+@Composable
+fun TimelineBottomView(
```

Comment:
The function name seems confusing. Would it be possible to name it `TimelineButtonView` since it only contains a button?